perf/istio-install/create_sa.sh

```diff
@@ -56,7 +56,3 @@ done
 gc projects add-iam-policy-binding "${PROJECT_ID}" --role "roles/meshconfig.writer" --member "serviceAccount:${GCP_CTL_SA}@${PROJECT_ID}.iam.gserviceaccount.com"
 # Required for creating the NEG objects in GCP
 gc projects add-iam-policy-binding "${PROJECT_ID}" --role "roles/compute.admin" --member "serviceAccount:${GCP_CTL_SA}@${PROJECT_ID}.iam.gserviceaccount.com"
-
-if [[ "${CLUSTER_NAME}" != "" ]]; then 
```

Comment:
@costinm removing this will break istiod right.

Reply:
Why does istiod require this? Seems like it shouldn't, unless we are doing something weird for test purposes? Just running this script triggers an email alerting me I downloaded a SA key.

Happy to add it back if we have a reason though